.github/workflows/build-deploy-pudl.yml

```diff
@@ -97,7 +101,6 @@ jobs:
             --container-env-file="./docker/.env" \
             --container-env ACTION_SHA=$ACTION_SHA \
             --container-env GITHUB_REF=${{ env.GITHUB_REF }} \
-            --container-env API_KEY_EIA=${{ secrets.API_KEY_EIA }} \
```

Comment:
It looks like the deployment got much further and may have had actual issues. Not sure if it's just because the expectations are based on having the fuel prices filled in, or if there are real issues to address. 

```
FAILED test/validate/eia_test.py::test_no_null_cols_eia[eia_raw-gf_eia923-all]
FAILED test/validate/eia_test.py::test_minmax_rows[eia_raw-gf_eia923-2662343-2662343-227945]
FAILED test/validate/eia_test.py::test_unique_rows_eia[eia_raw-gf_eia923-unique_subset4]
FAILED test/validate/gf_eia923_test.py::test_fuel_for_electricity[eia_raw] - ...
FAILED test/validate/gf_eia923_test.py::test_vs_bounds[eia_raw-coal_heat_content]
FAILED test/validate/gf_eia923_test.py::test_vs_bounds[eia_raw-oil_heat_content]
FAILED test/validate/gf_eia923_test.py::test_agg_vs_historical[eia_annual] - ...
FAILED test/validate/gf_eia923_test.py::test_agg_vs_historical[eia_monthly]
```

Reply:
Ah that makes sense. Do these tests depend on the fuel imputations?

Reply:
I don't **think** they intrinsically depend on the fuel price imputations, but they were certainly developed with the fuel prices being filled in, so they may have some implicit assumptions.